tests/sentry/api/test_utils.py

```diff
@@ -253,3 +255,25 @@ def test_handle_query_errors(self, mock_parse_error):
                     raise ex
             except Exception as e:
                 assert isinstance(e, (FooBarError, APIException))
+
+
+class IdOrSlugPathParamsEnabledTest(TestCase):
+    def test_no_options_enabled(self):
+        assert not id_or_slug_path_params_enabled("TestEndpoint.convert_args")
+
+    @override_options({"api.id-or-slug-enabled": True})
+    def test_ga_option_enabled(self):
+        assert id_or_slug_path_params_enabled(qual_name="TestEndpoint.convert_args")
+
+    @override_options({"api.id-or-slug-enabled-ea-org": ["sentry"]})
+    @override_options({"api.id-or-slug-enabled-ea-endpoints": ["TestEndpoint.convert_args"]})
+    def test_ea_org_option_enabled(self):
```

Comment:
Could you add one more test where org is enabled, but the convert_args isn't in the second option